docs/user/autosave.rst

```diff
@@ -0,0 +1,56 @@
+.. _autosave:
+
+Autosave in JupyterLab
+======================
+
+Every time you stop a JupyterLab notebook, a webhook is triggered and a
```

Comment:
Maybe we can simplify the phrasing here even a bit more - end users don't necessarily know what a webhook is.  Something like: 

"When you stop a JupyterLab notebook, an automatic check looks for any work that has not been pushed to the remote repository, including untracked and modified files."